scheduler/src/cook/kubernetes/api.clj

```diff
@@ -394,16 +404,33 @@
                                                      false)
              :else true))))
 
+(defn merge-resource-maps
+  "Given a function and two resource-maps, merge into a single resource map"
+  [f resource-map-a resource-map-b]
+  {:cpus (f (:cpus resource-map-a) (or (:cpus resource-map-b) 0))
+   :mem (f (:mem resource-map-a) (or (:mem resource-map-b) 0))
```

Comment:
Do we not need the `(or 0)` on `resource-map-a`?

Reply:
I don't think it's likely that the `or 0` would be used on `resource-map-a` but I can add it in just in case it is needed

Reply:
that's really helpful scott thanks!